ckan/lib/i18n.py

```diff
@@ -127,19 +127,19 @@ def _set_lang(lang):
     if config.get('ckan.i18n_directory'):
         fake_config = {'pylons.paths': {'root': config['ckan.i18n_directory']},
                        'pylons.package': config['pylons.package']}
-        i18n.set_lang(lang, pylons_config=fake_config, class_=Translations)
-    else:
-        i18n.set_lang(lang, class_=Translations)
-
 
+        try:
+            i18n.set_lang(lang, pylons_config=fake_config, class_=Translations)
+        except i18n.LanguageError:
```

Comment:
We were not capturing and ignoring these before, is this needed by the new behaviour (supporting `en`)?